components/raftstore/src/store/worker/read.rs

```diff
@@ -6,6 +6,7 @@ use std::{
     fmt::{self, Display, Formatter},
     ops::Deref,
     sync::{
+        atomic,
```

Comment:
Should be `atomic::{self, AtomicU64, Ordering}`.

Reply:
Good catch.